packages/nitro-protocol/test/contracts/NitroAdjudicator/concludePushOutcomeAndTransferAll.test.ts

```diff
@@ -166,23 +164,20 @@ describe('concludePushOutcomeAndTransferAll', () => {
       const largestTurnNum = turnNumRecord + 1;
       const initialFingerprint = ethers.constants.HashZero;
 
-      // Transfer some tokens into ERC20AssetHolder
+      // Transfer some tokens into the relevant AssetHoldera
       // Do this step before transforming input data (easier)
       if ('ERC20' in heldBefore) {
-        await (
-          await Token.transfer(ERC20AssetHolder.address, BigNumber.from(heldBefore.ERC20.c))
-        ).wait(); // if the tx is mined, we know we the transfer succeeded
+        await (await Token.increaseAllowance(ERC20AssetHolder.address, heldBefore.ERC20.c)).wait();
```

Comment:
Curious, why the switch from `transfer` to `increaseAllowance` + `deposit`?

Reply:
In the previous pattern we would simply pay tokens into the contract and then manually `setHoldings` to end up in a state where a given channel was funded with ERC20 tokens. That `setHoldings` method is one of the trapdoor, "test only"  (or "god mode") functions that I am removing in this PR. 

The `increaseAllowance` + `deposit` achieves the same end goal, but via the proper "production" route (API) that will actually be used in the wild.